Waiting for https://github.com/dotnet/BlazorMinifiedJs/pull/5 to be merged to fix
```
Comparing contents for blazor.web.js
Comparing contents for blazor.server.js
Comparing contents for blazor.webview.js
Diff found in blazor.web.js, please see https://github.com/dotnet/aspnetcore/blob/main/docs/UpdatingMinifiedJsFiles.md for remediation steps
Diff found in blazor.server.js, please see https://github.com/dotnet/aspnetcore/blob/main/docs/UpdatingMinifiedJsFiles.md for remediation steps
##[error]PowerShell exited with code '1'.
```

--- a/src/Mvc/test/Mvc.FunctionalTests/RazorRuntimeCompilationHostingStartupTest.cs
+++ b/src/Mvc/test/Mvc.FunctionalTests/RazorRuntimeCompilationHostingStartupTest.cs
@@ -85,6 +85,7 @@ public class RazorRuntimeCompilationHostingStartupTest : LoggedTest
     }
 
     [Fact]
+    [QuarantinedTest("https://github.com/dotnet/aspnetcore/issues/56553")]
     public async Task RazorPages_CanBeServedAndUpdatedViaRuntimeCompilation()
     {
         // Arrange

